backend/ml_model/repository/safe_test.py

```diff
@@ -0,0 +1,36 @@
+from sklearn.model_selection import train_test_split, GridSearchCV
+from sklearn.tree import DecisionTreeClassifier
+
+
```

Comment:
Why arent we making this a class?

Reply:
I think I left it as functions as we did with the model_saver files because they're necessary functions but not really a necessary class?